requirements.sh

```diff
@@ -72,13 +72,16 @@ if [ ! -d "${directory}/opendnp3" ]; then
 	git clone --recursive -b release https://github.com/dnp3/opendnp3.git
 	cd opendnp3
 	# Until we hve a newer libasio on all the platforms we support we will
-	if [[ ${OS_NAME} == *"Ubuntu"* && ${OS_VERSION} == *"20"* ]]; then
+	if [[ ${OS_NAME} == *"Ubuntu"* && ${OS_VERSION} == *"20."* ]]; then
 		# stick with release 2.3.0 of opendnp3
 		git checkout tags/2.3.0
 		# Set TLS flag
 		useTLS=1
+        elif [[ ${OS_NAME} == *"Ubuntu"* && ( ${OS_VERSION} == "22."* || ${OS_VERSION} == "24."* ) ]]; then
+	        # Set TLS flag
```

Comment:
is it not a good idea to stick a fixed tag ..?

Reply:
Ideally Yes.

Looks like there were compatibility issues hence this approach.

Though I can see this can be improved and tested with newer stable release.

Bullseye Raspberry Pi OS also can have TLS.